android/src/main/java/com/twiliorn/library/TwilioRemotePreview.java

```diff
@@ -23,10 +23,6 @@ public TwilioRemotePreview(ThemedReactContext context, String trackSid) {
         Log.i("CustomTwilioVideoView", "Remote Prview Construct");
         Log.i("CustomTwilioVideoView", trackSid);
 
-        CustomTwilioVideoView.registerPrimaryVideoView(this.getSurfaceViewRenderer(), trackSid);
-    }
-
-    public void applyZOrder(boolean applyZOrder) {
```

Comment:
applyZOrder isn't needed for texture view and so I removed it since we don't have a separate path to use the surface view